zipkin-autoconfigure/metrics-prometheus/README.md

```diff
@@ -0,0 +1,18 @@
+# Prometheus Metrics
```

Comment:
should probably make a link from zipkin-server's README to this, under the metrics section. Saying something like: Zipkin also exports metrics in prometheus format (link)